### PR TITLE
chore: Prepare for Release 0.14.0

### PR DESCRIPTION
**Pull request by miquelbeltran**
### Description :memo:
- **Purpose**: Prepare package for release
- **Approach**: Bump version number and update changelog.

**Type of change**

- [x] Release

**Updates**

- Sets version to `0.14.0`
- Adds Changelog.

### Author to check :eyeglasses:
- [ ] Project and all contained modules builds successfully
- [ ] Self-/dev-tested 
- [ ] Unit/UI/Automation/Integration tests provided where applicable
- [ ] Code is written to standards
- [ ] Appropriate documentation written (code comments, internal docs)

### Reviewer to check :heavy_check_mark:
- [x] Project and all contained modules builds successfully
- [x] Change has been dev-/reviewer-tested, where possible
- [x] Unit/UI/Automation/Integration tests provided where applicable
- [x] Code is written to standards
- [x] Appropriate documentation written (code comments, internal docs)

### CODE REVIEW

**Comment by miquelbeltran** [comment]
As discussed internally, this contains the preparations for the 0.14.0 release.

The release won't be done until we merge to `master`, this only prepares the package and the changelog.

File: package.json
Comment (sumitramanga) [inline package.json:3]:
```suggestion
  "description": "Raygun package for Node.js, written in TypeScript",
```

File: package.json
Comment (sumitramanga) [inline package.json:1]: It would be good to fill in the keywords array with common search terms like 'Error reporting' whilst this exists! Seems like this will help with discovery within the npm registry :)

**Comment by sumitramanga** [comment]
Should we update the license within the main README to 2024? It's currently at 2016

**Comment by sumitramanga** [comment]
Nit pick - In the README, there's a link like this - http://raygun.io/blog/2013/07/video-nodejs-error-handling-with-raygun/
When it should be https://raygun.com/blog/2013/07/video-nodejs-error-handling-with-raygun/

It's a pretty old blog but might pay to just correct the URL

**Comment by sumitramanga** [comment]
Testing went well! Was able to receive errors. I was about to test Real User Monitoring but it looks like Node is not built for that product (I tried a simple test and couldn't get it to work) 

I tested - 
- Both example apps 
- Ran all the commands in the package.json for test suites, prettier, eslint and prepare
- Installed this version (0.14.0) of raygun4node within a brand new Node app 

I'm ready to approve this PR

File: package.json
Comment (miquelbeltran) [reply package.json:1]: Added them, feel free to modify the array as you think!

**Comment by miquelbeltran** [comment]
@sumitramanga I did the requested changes.

File: package.json
Comment (TheRealAgentK) [inline package.json:3]: Might be worthwhile to mention bun as well given the traction it seems to get?

File: package.json
Comment (TheRealAgentK) [inline package.json:73]: Some more ideas (they don't have to be single words, I think)

crash reporting
error tracking
raygun
APM

Comment (miquelbeltran) [reply package.json:73]: seems to be a mix of different formats: https://www.npmjs.com/search?q=crash

e.g. `crash` `reporting` `crash-reporting` `crash reporting`

Comment (miquelbeltran) [reply package.json:73]: If you search for "crash reporting" you already get results with packages with them as separated words. I'd stick to that.

image

Comment (TheRealAgentK) [reply package.json:73]: debug
logging

might be relevant as well.

**Review by TheRealAgentK — APPROVED** [review]

**Comment by TheRealAgentK** [comment]
I've approved already because I think the release as such is good, pending whatever comes out of the keyword discussion. But that's really just additional findability optimisation.

**Comment by miquelbeltran** [comment]
I'll wait for the approval from @sumitramanga, and then I'll open the "rebase to master" in a new PR once this is merged into `develop`.

File: package.json
Comment (sumitramanga) [reply package.json:1]: Those key words look great! Thank you

**Review by sumitramanga — APPROVED** [review]
LGTM :shipit:

**Comment by sumitramanga** [comment]
Had a chat with Miguel, let's just merge this in now and get Panos involved for the release PR next up

**Pull request merged**
